src/robusta/core/model/env_vars.py

```diff
@@ -33,6 +33,7 @@
 PROMETHEUS_REQUEST_TIMEOUT_SECONDS = float(
     os.environ.get("PROMETHEUS_REQUEST_TIMEOUT_SECONDS", 90.0)
 )
+PROMETHEUS_ENABLED = os.environ.get("PROMETHEUS_ENABLED", False)
```

Comment:
os.environ.get returns str, and the default value here is bool

Might be safer to do something like:
PROMETHEUS_ENABLED = os.environ.get("PROMETHEUS_ENABLED", "false").lower() == "true"